libraries/microlibrary/ANY/ANY/include/microlibrary/integer.h

```diff
@@ -47,6 +48,27 @@ constexpr auto to_unsigned( Integer integer ) noexcept
     return unsigned_integer;
 }
 
+/**
+ * \brief Lookup an unsigned integer's highest bit set.
+ *
+ * \tparam Integer The type of unsigned integer the lookup will be performed on.
+ *
+ * \param[in] integer The unsigned integer to perform the lookup on.
+ *
+ * \attention This function assumes integer is non-zero.
+ *
+ * \return The integer's highest bit set.
+ */
+template<typename Integer>
+constexpr auto highest_bit_set( Integer integer ) noexcept -> std::uint_fast8_t
```

Comment:
Replace `integer` with a better parameter name (maybe `value`). Will need to update automated tests as well.